spec/support/expire_cookies.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+module ExpireCookies
+  def expire_cookies
```

Comment:
a) You're testing the wrong functionality — instead of _actually passing time_ (or properly mocking `Time.now` or whatever) you just hack out that behavior from your program and replace it with something else.
b) I'm not sure why you should care about number and size of gems which are not going to your `production` environment. For production — that's a fairly good concern, but for testing purposes — meh.